lr-cli/cortx_setup/commands/node/finalize.py

```diff
@@ -215,6 +215,29 @@ def _create_field_users(self, node_id):
                     targets=node_id
                 )
 
+            else:
+                self.logger.debug(f"Ensuring {user} user permissions on system")
+
+                StateFunExecuter.execute(
+                    'file.append',
+                    fun_kwargs=dict(
+                        name=f'{home_dir}/.bashrc',
```

Comment:
Can we add alias to profile instead? Lets not make it bash specific

Reply:
Alias for specific commands in profile doesn't work.